src/Symfony/Component/Translation/DependencyInjection/TranslatorPass.php

```diff
@@ -19,13 +19,17 @@
 class TranslatorPass implements CompilerPassInterface
 {
     private $translatorServiceId;
-    private $loaderServiceId;
+    private $readerServiceId;
     private $loaderTag;
 
-    public function __construct($translatorServiceId = 'translator.default', $loaderServiceId = 'translation.loader', $loaderTag = 'translation.loader')
+    public function __construct($translatorServiceId = 'translator.default', $readerServiceId = 'translation.loader', $loaderTag = 'translation.loader')
     {
+        if ('translation.loader' === $readerServiceId) {
+            @trigger_error('The default value for $readerServiceId will change in 4.0 to "translation.reader".', E_USER_DEPRECATED);
```

Comment:
Indeed, we need to mute the deprecation for internal uses by changing this line https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php#L89 to:

```diff
- $this->addCompilerPassIfExists($container, TranslatorPass::class);
+ if (class_exists(TranslatorPass::class)) {
+     $container->addCompilerPass(new TranslatorPass('translator.default', 'translation.loader', 'translation.loader', false);
+ }
```

and change this check:
```diff
- if ('translation.loader' === $readerServiceId) {
+ if ((4 > func_num_args() || false !== func_get_arg(3)) && 'translation.loader' === $readerServiceId) {
```
That should fix tests (deps=high can remain red because the code is not there on master).